### ISSUE TITLE
Adding PNG files to animate makes the animation darker as I progress

### ISSUE THREAD

**Issue by MUI68K** [alexzielenski/Mousecape#113]
I've begun making a new cursor set for Mousecape and now with macOS Catalina 10.15.4 there's a big that I can't wrap my head around.. Instead of adding my files instead a strange thing happen..

Help? I have been using Sketch 64 - upgraded from I think 61 when I made the Ignite mousecape set. 
 
Look at the attached image and you can sort of get an idea about what I'm ranting about.

There's  desktop recording here.. atleast for a while .. showing the thing.
[https://jumpshare.com/v/swAlLIlSCHwtuw7J10Jj+/Screen+Recording+2020-04-02+at+23.05.55.mov](url)
![Screen Shot 2020-04-02 at 23 08 00](https://user-images.githubusercontent.com/60530401/78300256-7d0ee280-7537-11ea-89a6-e13aa2267193.png)

**Comment by jsteweii** [comment]
I'm not on Catalina yet, but from what I see from the screenshot it seems that the 24 images get stacked on top of each other. So instead of an animation, you just get one non animated image where the white part gets more grey because of the stacking. Just my two cents:-).

**Comment by MUI68K** [comment]
> I'm not on Catalina yet, but from what I see from the screenshot it seems that the 24 images get stacked on top of each other. So instead of an animation, you just get one non animated image where the white part gets more grey because of the stacking. Just my two cents:-).

that's what's happening yes - I have to dig deeper because even if I get the animation to roll I still see issues.. I can't fathom it. I'll see if Sketch is doing something to my files when exporting or maybe it's a plugin in Sketch 64 that's bad.

**Comment by jsteweii** [comment]
If you want, I can have a look at it. At least to confirm if it's on your end or more generel.

**Comment by MUI68K** [comment]
> If you want, I can have a look at it. At least to confirm if it's on your end or more general. I don't use Sketch though.

Thanks - if you would - try just adding in from "WheelAnimate_0" - the animation isn't complete and will eventually be replaced with the other one I have in the works - this is just to test.
For anyone to test, really,as  it's free to download the whole archive should anyone care for it.

The cape is included. If you would try importing that and start from there? 👍 
[luxuscape-test.zip](https://github.com/alexzielenski/Mousecape/files/4436964/luxuscape-test.zip)

**Comment by jsteweii** [comment]
Ok, so I don't see the issue on Mojave. I'm on latest Mousecape build and 10.14.6 Mojave. I'm using Gimp to test.

**Comment by MUI68K** [comment]
@jsteweii yeah, that doesn't have any issues with what I previously complained about.. odd.
Thanks for making this and investigating - I'll try something different for my next try when it appears that what I have previously done doesn't work.
